### ISSUE TITLE
selected state of wizard options is not accessible to screen readers

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#700]
It looks like the aria-selected property is the likely solution here: https://www.w3.org/TR/wai-aria/states_and_properties#aria-selected
